packages/xrpc-server/src/stream/index.ts

```diff
@@ -3,3 +3,4 @@ export * from './frames'
 export * from './stream'
 export * from './subscription'
 export * from './server'
+export * from './websocket-keepalive'
```

Comment:
Let's get a changeset in for this so the published version of `@atproto/ozone` is able to use the functionality.

Reply:
do i need to do that separately or just including `@atproto/xrpc-server` in the changeset of this PR would take care of that?

Reply:
I would create two changesets in this PR: the existing one for "Add verification lexicons to ozone" and then a second one just for xrpc-server describing the change there.  That way in the xrpc-server changelog it will talk about the change relevant to that package rather than talking about verifications.